.ci/docker.sh

```diff
@@ -28,6 +28,10 @@ while [[ $# != 0 ]]; do
       GET=1
       shift
       ;;
+    '--interactive')
```

Comment:
I have put this change into its own pr as well https://github.com/Cockatrice/Cockatrice/pull/4610